code/game/turfs/simulated/floor.dm

```diff
@@ -51,8 +51,8 @@
 		set_flooring(get_flooring_data(floortype), TRUE)
 		. = INITIALIZE_HINT_LATELOAD // We'll update our icons after everyone is ready
 	else
```

Comment:
If you remove an else case, make sure to remove the entire section and don't leave it standing alone

Reply:
I may be dumb